section2/exercises/methods.rb

```diff
@@ -5,26 +5,40 @@
 
 # Example: Write a method that when called will print your name:
 def print_name
-  p "Severus Snape"
+  puts "Severus Snape"
 end
 
 print_name
 
 # Write a method that takes a name as an argument and prints it:
 def print_name(name)
-  # YOUR CODE HERE
+  puts "#{name}"
 end
 
 print_name("Albus Dumbledore")
 
-# Write a method that takes in 2 numbers as arguments and prints 
+# Write a method that takes in 2 numbers as arguments and prints
 # their sum. Then call your method three times with different arguments passed in:
-# YOUR CODE HERE
+def sum_of_two_numbers(num_1, num_2)
+  puts num_1 + num_2
+end
+
+sum_of_two_numbers(1, 3)
+sum_of_two_numbers(0, 45)
+sum_of_two_numbers(92, -2)
+
+# Write a method that takes in two strings as arguments and prints
+# a concatenation of those two strings. Example: The arguments could be
+# (man, woman) and the end result might output: "When Harry Met Sally".
+# Then call your method three times with different arguments passed in.
+
+def get_coffee_with_friends(friend_1, friend_2)
+  puts "#{friend_1} and #{friend_2} get coffee at Central Perk."
```

Comment:
Check out the difference between string interpolation and string concatenation :)